src/foam/core/servlet/VirtualHostRoutingServlet.js

```diff
@@ -188,8 +194,9 @@ foam.CLASS({
       }
 
       // Loading screen styles
+      out.println("<meta name=\\"csp-nonce\\" content=\\"" + getCspNonce() + "\\">");
+      out.println("<style nonce=\\"" + getCspNonce() + "\\">");
```

Comment:
This should only be output when non-null. 
The cspNonce default should be null. 
Then in the PT services.jrl you can add one and then only PT deployments will have a nonce value.